meta-balena-common/recipes-core/os-config/os-config_1.2.6.bb

```diff
@@ -251,9 +262,12 @@ SRC_URI += " \
     crate://crates.io/winreg/0.6.2 \
     crate://crates.io/winutil/0.1.1 \
     crate://crates.io/ws2_32-sys/0.2.1 \
+    git://github.com/balena-os/fatrw;protocol=https;nobranch=1;name=fatrw;destsuffix=fatrw \
 "
 
-
+SRCREV_FORMAT .= "_fatrw"
+SRCREV_fatrw = "${AUTOREV}"
```

Comment:
Could we use a commit SHA1 instead of AUTOREV? With `AUTOREV` it's impossible to tell what version was actually built.

Reply:
Nice catch. Looking at the code of `cargo-bitbake` this is coming from autogeneration done from reading this part of os-config's Cargo.toml:
```
[dependencies.fatrw]
git = "https://github.com/balena-os/fatrw"
```

I will pin it to specific `fatrw` version and not to what is latest on master.

Reply:
@alexgg done now. I was able to use the tag for the particular release. We can use the commit SHA1 as well, but I thought using the tag instead fits better the way other dependencies are specified in Cargo.toml: https://github.com/balena-os/os-config/blob/master/Cargo.toml#L42-L44. Please let me know if you think using the SHA1 will be more suitable though.

Reply:
@majorz With the update to Kirkstone tags will not work (as tags can be removed and re-created) and the build will fail with an error. It needs to be a sha1.

Reply:
@alexgg sounds good!